README.md

```diff
@@ -33,6 +35,49 @@ Command:
 Example:    
 `python fix-host-save.py "C:\Users\John\.cargo\bin\uesave.exe" "C:\Users\John\Desktop\my_temporary_folder\2E85FD38BAA792EB1D4C09386F3A3CDA" 6E80B1A6000000000000000000000000 00000000000000000000000000000001`
 
+### Change Co-op host
+
+Prepare:
+
+Fill `uesave_path`、`save_path` and `player_list` in *config.json*, for example:
+
+```json
+{
+  "uesave_path": "C:\\Users\\John\\.cargo\\bin\\uesave.exe",
+  "save_path": "C:\\Users\\John\\Desktop\\my_temporary_folder\\2E85FD38BAA792EB1D4C09386F3A3CDA",
+  "player_list": [
+    {
+      "GUID": "6E80B1A6000000000000000000000000",
+      "name": "Arthur"
```

Comment:
I modified the script and dumped .sav to .json, then I found there's no player name inside the save file. How do I identify which player owns the save file?

Reply:
This "name" serves an internal purpose within the script, primarily to simplify the command for running the script and make it more memorable. The exported JSON files still rely on the GUID to distinguish players.